src/test/scala/com/softwaremill/realworld/articles/CommentsSpecData.scala

```diff
@@ -0,0 +1,178 @@
+package com.softwaremill.realworld.articles
+
+import com.softwaremill.realworld.articles.comments.{Comment, CommentCreate, CommentCreateData, CommentData, CommentsList}
+import com.softwaremill.realworld.profiles.ProfileData
+import com.softwaremill.realworld.utils.TestUtils.backendStub
+import sttp.client3.{Response, ResponseException, basicRequest}
+import sttp.client3.ziojson.asJson
+import sttp.model.Uri
+import sttp.tapir.json.zio.jsonBody
+import zio.ZIO
+import zio.test.Assertion.{equalTo, isRight}
+import zio.test.{TestResult, assertTrue, assertZIO}
+import sttp.client3.ziojson.zioJsonBodySerializer
+
+import java.time.Instant
+
+object CommentsSpecData {
+
+  def zioEffect(
```

Comment:
we should be able to get a better name; zio effect is a technical thing, what does this method do?

Reply:
It's just a request that we use in tests. I changed the names to more specific ones, for example. "getCommentsFromArticleRequest", "addCommentRequest"

Reply:
or I will do it as Krzysiek wrote in the comment below.